app/javascript/components/pages/PatientInfo.js

```diff
@@ -45,20 +45,43 @@ export default class PatientInfo extends Component {
     } = this.state.patient;
     const { patient } = this.state;
     return (
-      <div>
-        <img src={patient.image} alt={patient.first_name} />
-        <h2>
-          {patient.first_name} {patient.last_name}
-        </h2>
-        {console.log("here", medications)}
-        {medications.map((medication, idx) => {
-          return (
-            <div key={idx}>
-              <p>{medication.medication_name}</p>
-            </div>
-          );
-        })}
-      </div>
+      <>
+        <div>
+          <img src={patient.image} alt={patient.first_name} />
+          <h2>
+            {patient.first_name} {patient.middle_name} {patient.last_name}
+          </h2>
+          <ul>
+            <li>Preferred name: {patient.preferred_name}</li>
+            <li>DOB: {patient.dob}</li>
+            <li>Gender: {patient.gender}</li>
+            <li>Pronoun: {patient.pronoun}</li>
+            <li>Need To Know: {patient.need_to_know}</li>
+          </ul>
+          <h3>Medications</h3>
+          {medications &&
+            medications.map((medication, idx) => {
+              return (
+                <div key={idx}>
+                  <p>{medication.medication_name}</p>
+                </div>
+              );
+            })}
+        </div>
+        <a href={`/patientedit/${this.props.id}`} className="backButton">
```

Comment:
All these `<a>` tags can be updated to use react routers [link component](https://v5.reactrouter.com/web/api/Link). You get some cool functionality with it (not that you're really using it here)

Reply:
Can do, will change to either a `Link` or a `Button`

Reply:
will discuss this with team